third_party/xxhash/BUILD

```diff
@@ -0,0 +1,11 @@
+cc_library(
```

Comment:
isn't it confusing that you're adding the BUILD file to a directory that's entirely imported as git subtree, but this is the only file that you manually contributed yourself?
I'd like to get a second opinion on this.  @apolcyn  @veblush  WDYT?

Reply:
Don't we have another way like having it as a submodule and placing our BUILD for it at third_party? Otherwise, I guess we should do like this?

Reply:
I tried putting it in third_party/xxhash.BUILD (which works if we were adding the dependency via http_archieve as we can specify a build file location); but since I am just doing it directly in grpc_build_system.bzl i don't have a way to specify the BUILD file location it assumes it is in the directory and named as "BUILD"